definitions/rest-definitions.d.ts

```diff
@@ -13,10 +13,12 @@ declare module "rest-definitions" {
 
     export interface DeploymentStatusReport {
         appVersion: string;
-        clientUniqueId: string;
-        deploymentKey: string; 
+        clientUniqueId?: string;
+        deploymentKey: string;
+        fromDeploymentKey?: string;
```

Comment:
I like "previous" more than "source", the "source" is the server. I'll change it all to "previous" then.